lib/african_poetics/index_person.rb

```diff
@@ -59,13 +70,11 @@ def spatial
     end
   end
 
-  def subcategory
-    if @record.major_african_poet
-      "Person (Poet)"
-    else
-      "Person (Non-Poet)"
-    end
-  end
+  # NOTE only poets should be getting indexed,
+  # so there is no need to distinguish which are
+  # major african poets here
+  # def subcategory
+  # end
```

Comment:
I think that it's falling back on the default behavior, `@record_type` because it's commented out. If it's empty, it would be nil. Basically all of the `index_blah.rb` files are just overriding the `fields.rb` file.  I can delete it if you want, I think I had left it in there because the comment above is relevant to that field.  I don't think it's important to keep, though, if that would alleviate confusion.

Reply:
Ah okay. No, this sounds fine then. I should have remembered the default/parent method because of the use of inheritance :nerd_face: